modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,5 +1,37 @@
 = Release Notes for Couchbase Server 7.2
 
+[#release-723]
+== Release 7.2.3 (November 2023)
+Couchbase Server Release 7.2.3 was released in November 2023
+
+[IMPORTANT]
+====
+Fix contains upgrades to fix the following advisories:
```

Comment:
@RayOffiah I just looked for an example of how we've identified security vulnerabilities in the past. I think you should model this page:  https://docs.couchbase.com/server/7.0/release-notes/relnotes.html#fixed-security-vulnerabilities